docs/sp-add-ins/deploy-and-install-a-sharepoint-hosted-sharepoint-add-in.md

```diff
@@ -65,7 +65,8 @@ As the site collection is being created, SharePoint takes you back to the **Shar
 
 4. Select **Trust It** in the consent dialog. The **Site Contents** page automatically opens and the add-in appears with a notation that it is installing. After it installs, users can select the tile to run the add-in.
 > [!Important]
-> it might take up to 24 hours for add-ins to be available for consumption once installed. During that time, users can recive a "The redirect URI specified in the request does not match the redirect URIs configured for the application" AADSTS50011 error.
+> it might take up to 24 hours for add-ins to be available for consumption once installed. During that time, users can recive a "The redirect URI specified in the request does not match the redirect URIs configured for the application" AADSTS50011 error. If you need to shortcut that time and you have administrative permissions in your tenant, you can add that URL to the Application mentioned in the error ('00000003-0000-0ff1-ce00-000000000000') 
+using [Microsoft Graph Powershell](https://learn.microsoft.com/en-us/powershell/microsoftgraph/overview?view=graph-powershell-1.0)

```

Comment:
@lucabandMSFT : think we should be include the sample script here

Reply:
Sample script to include can be this:

```PowerShell
Connect-MgGraph -Scopes Application.ReadWrite.All

$sharepointPrincipal = Get-MgServiceprincipal -Filter "AppId eq '00000003-0000-0ff1-ce00-000000000000'"

$sharepointPrincipal | fl

$replyUrls = $sharepointPrincipal.ReplyUrls

$replyUrls += "https://contoso-79d42f062409ae.sharepoint.com/_forms/default.aspx"

Update-MgServiceprincipal -ServicePrincipalId $sharepointPrincipal.Id -ReplyUrls $replyUrls
```

Reply:
I fixed a few things in the doc, but I'll let you guys decide on the script that @jansenbe referenced so I don't inject myself into the process...

Reply:
@andrewconnell, @jansenbe : question for both of you, if I include the script, is it ok to use a fake tenant name like contoso.sharepoint.com? Also, if I do that, should I include a full example of an error with the fake (contoso) name as the error will have both the name of the tenant and the appID?

Reply:
@lucabandMSFT said:
> if I include the script, is it ok to use a fake tenant name like contoso.sharepoint.com

Yup, in fact that's the exact one I'd use

> if I do that, should I include a full example of an error with the fake (contoso) name as the error will have both the name of the tenant and the appID?

If you think it would be helpful, sure... but it can also be noise in the doc if it's a long error snippet. My approach is to only add it if it adds value, but to err on not adding noise. If the doc gets questions or warrants more detail, then sure. Then again, it can't hurt to add it... but maybe add it as an Appendix at the end of the doc?

Reply:
Thanks @andrewconnell. Let me rewrite it in a bit.

Reply:
edits... waiting for the automated build to complete to verify formatting looks correct... will merge if all is good